experimental/dds/tree2/src/class-tree/schemaTypes.ts

```diff
@@ -131,9 +148,23 @@ export enum FieldKind {
  * @beta
  */
 export enum NodeKind {
+	/**
+	 * Node which serves as a map, storing children under string keys.
```

Comment:
Nit: I'm arguing for complete sentences in our public facing docs where possible for accessibility. So I would recommend rephrasing these as `A node which...`

Reply:
```suggestion
	 * A node which serves as a map, storing children under string keys.
```